### ISSUE TITLE
Training Loss Abnormal

### ISSUE THREAD

**Issue by Haoyanlong** [Wendison/VQMIVC#35]
@andreasjansson @Wendison Hello, sorry to interrupt you!  I'm a rookie of voice model. I have trained the model in VCTK-Corpus-0.92.zip dataset by "python3 train.py use_CSMI=True use_CPMI=True use_PSMI=True" in NVIDIA V100S. But after 65 epochs, the train loss are as follows:
![image](https://user-images.githubusercontent.com/23288056/178647127-f58b2e58-a397-4625-a59b-698ff94829e3.png)
Could you give me some advice? Thank you very much!

**Comment by Wendison** [comment]
Hi, I think the lld losses are normal, you could train for more epoches and listen to converted samples to verify whether your training is successful.

**Comment by Haoyanlong** [comment]
@Wendison Hello! I have two questions:
1. The default epoch of training is 500. I have achieved the process, training log as folllows(500 epochs).  Whether each loss value meets expectations?
![image](https://user-images.githubusercontent.com/23288056/178683682-88a38daf-57c4-4576-9676-37caed17b87d.png)

2. I want to see the training effect quickly.How many epochs do you suggest to train?
Thank you!

**Comment by Wendison** [comment]
1. I don't remember the exact value of each loss, but I think your losses should be normal according to the losses shown in https://github.com/Wendison/VQMIVC/issues/15#issue-1025051309
2. Based on my experience, 500 epoches can obtain stable conversion results. After 500 epoches, I didn't see any noticable performance improvements. One direct way to select the suitable epoch is to listen to intermediate converted samples during training.
